Packs/CloudConvert/Integrations/CloudConvert/CloudConvert.py

```diff
@@ -0,0 +1,377 @@
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+
+    @logger
+    def __init__(self, headers, verify=False, proxy=False):
+        url = 'https://api.cloudconvert.com/v2'
+        super().__init__(url, headers=headers, verify=verify, proxy=proxy)
+    
+    @logger
+    def import_url(self, arguments: Dict[str, Any]) -> Dict[str, Any]:
+        """
+        Import the file given as url to the API's server, for later conversion
+
+        :type arguments: ``Dict[str, Any]``
+        :param arguments: dict containing the request arguments, should contain the field 'url'
+        :return: dict containing the results of the import action as returned from the API (status, file ID, etc.)
+        :rtype: ``Dict[str, Any]``
+        """
+        return self._http_request(
+            method='POST',
+            url_suffix='import/url',
+            headers=self._headers,
+            data=arguments,
+            timeout=100,
+            ok_codes=(422, 200, 201, 500, 401)
+        )
+    
+    @logger
+    def import_entry_id(self, file_path: str, file_name: str) -> Dict[str, Any]:
+        """
+        Import the file given as a war room entry id to the API's server, for later conversion
+
+        :param file_path: path to given file, derived from the entry id
+        :param file_name: name of file, including format suffix
+        :return: dict containing the results of the import action as returned from the API (status, file ID, etc.)
+        :rtype: ``Dict[str, Any]``
+        """
+        response_get_form = self._http_request(
+            method='POST',
+            url_suffix='import/upload',
+            timeout=100,
+            headers=self._headers
+        )
+        form = response_get_form.get('data', {}).get('result', {}).get('form', {})
+        port_url = form['url'] # if no url field in form, we should have an exception
+        params = form['parameters']
+
+        # Creating a temp file with the same data of the given file
+        # This way the uploaded file has a path that the API can parse properly
+        # The reason is that the API asserts the file type by the file extension,
+        # and getFilePath returns a file name without that extension
+        demisto.debug('creating a temp file for upload operation')
+        with tempfile.TemporaryFile(suffix=file_name) as temp_file:
+            with open(file_path, 'rb') as file:
+                temp_file.write(file.read())
+            temp_file.seek(0)
+            file_dict = {file_name: temp_file}
+
+            self._http_request(
+                method='POST',
+                url_suffix=None,
+                full_url=port_url,
+                headers=self._headers,
+                files=file_dict,
+                empty_valid_codes=[201, 204],
+                return_empty_response=True,
+                timeout=100,
+                data=params
+            )
+
+        response_get_form['data'].pop('message')
```

Comment:
im popping those values because they are relevant to the first request (the one that is used to get the form), thus have values that are not true after the second request (for example, it has a message saying it is waiting for a file upload - which is already being done on the second request). I will update the default value :)